scripts/speaker_tasks/pathfiles_to_diarize_manifest.py

```diff
@@ -126,7 +126,7 @@ def main(
 
         duration = None
         if add_duration:
-            y, sr = librosa.get_duration(filename=audio_line, sr=None)
+            y, sr = librosa.load(audio_line, sr=None)
```

Comment:
Could you explain why this line is included in this PR?

Reply:
librosa removed reading sr from get_duration function